packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -1348,6 +1403,57 @@ describe("normalizeAndValidateConfig()", () => {
   });
 
   describe("named environments", () => {
+    it("should warn if we specify an environment but there are no named environments", () => {
+      const rawConfig: RawConfig = {};
+      const { diagnostics } = normalizeAndValidateConfig(
+        rawConfig,
+        undefined,
+        "DEV"
+      );
+      expect(diagnostics.renderErrors()).toMatchInlineSnapshot(`
+        "Processing wrangler configuration:
+        "
+      `);
+      expect(diagnostics.renderWarnings()).toMatchInlineSnapshot(`
+        "Processing wrangler configuration:
+          - No environment found in configuration with name \\"DEV\\".
+            Before using \`--env=DEV\` there should be an equivalent environment section in the configuration.
+
+            Consider adding an environment configuration section to the wrangler.toml file:
+            \`\`\`
+            [env]
+            DEV = { }
```

Comment:
the docs all use the `[env.DEV]` format, maybe we should stick with that 
```suggestion
            [env.DEV]
```

Reply:
This is what `TOML.stringify()` produces... Can we configure that? Or should I give up with `TOML` and hand craft this?

Reply:
for this line, go ahead and handcraft it I think

Reply:
I went with handcrafted, since it is so simple.